Experience/experience.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Resume & Honors!
+		</title>
+		<link rel="stylesheet" type="text/css" href="../style.css">
+	</head>
+	
+	<body>
+
+		<nav>
+			<a href="../index.html"> Home </a> |
+			<a href="experience/experience.html" target="_blank"> Resume and Honors</a> |
```

Comment:
:hammer: This link isn't working. Looks like something is lowercase here that needs to be uppercase. Can you see which one? Remember, `directory/somefile.html` is _totally_ different than `Directory/SomeFile.html`. Case matters.